### PR TITLE
Add rest api settings

### PR DESCRIPTION
**Pull request by eloaverona**
This PR adds a ui for the user to set the Sawtooth rest api settings:

### CODE REVIEW

File: examples/xo_android_client/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,15 +23,22 @@
...
+                android:label="@string/title_activity_settings"
+                android:parentActivityName=".MainActivity">
+            <meta-data
+                    android:name="android.support.PARENT_ACTIVITY"
+                    android:value="io.bitwise.sawtooth_xo.MainActivity"/>
+        </activity>
     </application>
 
-</manifest>
+</manifest>
```
Comment (shannynalayna) [inline examples/xo_android_client/app/src/main/AndroidManifest.xml]: newline

File: examples/xo_android_client/app/src/main/res/xml/preferences.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <android.support.v7.preference.EditTextPreference
+            android:key="@string/rest_api_settings_key"
+            android:title="@string/rest_api_settings_title"
+            android:summary="@string/rest_api_settings_summary"
+            android:dialogMessage="@string/rest_api_settings_dialog_message"
+            android:defaultValue="@string/default_rest_api_address" />
+
+</PreferenceScreen>
```
Comment (shannynalayna) [inline examples/xo_android_client/app/src/main/res/xml/preferences.xml]: newline

File: examples/xo_android_client/app/src/main/res/menu/list_view_menu.xml [hunk 3]
```diff
@@ -6,4 +6,8 @@
             android:title="@string/refreshBtn"
             app:showAsAction="ifRoom"/>
 
+    <item
+            android:id="@+id/settings"
+            android:title="@string/SettingsBtn"
+            app:showAsAction="withText|never"/>
 </menu>
```
Comment (shannynalayna) [inline examples/xo_android_client/app/src/main/res/menu/list_view_menu.xml]: newline

**Review by shannynalayna — CHANGES_REQUESTED** [review]
Looks like a few of the files need newlines at the end, otherwise it looks good to me!

**Review by shannynalayna — APPROVED** [review]
lgtm

**Review by dplumb94 — APPROVED** [review]

**Pull request merged**